src/main/resources/config.properties

```diff
@@ -21,4 +22,11 @@ build.workspace.computer.not.found = Unable to access the computer for this buil
 matlab.command.build.step.name = runMATLABCommand
 matlab.tests.build.step.name = runMATLABTests
 matlab.command.step.display.name = Run MATLAB commands, scripts, or functions
-matlab.tests.step.display.name = Run MATLAB tests and generate artifacts
\ No newline at end of file
+matlab.tests.step.display.name = Run MATLAB tests and generate artifacts
+matlab.custom.location = Custom...
+matlab.added.to.path.from = Prepending MATLAB to the PATH variable from this location: %s
+Axis.matlab.key = MATLAB
+Axis.use.matlab.warning = This project specifies MATLAB versions using the added 'MATLAB' axis as well as the 'Use MATLAB version' option. The value specified by 'Use MATLAB version' take precedence over the values specified by the 'MATLAB' axis.
+Axis.no.installed.matlab.error = Because no MATLAB versions are configured in Jenkins Global Tool Configuration, the plugin will run the default matrix configuration.
```

Comment:
Because no MATLAB versions are configured in Jenkins Global Tool Configuration,

Let's replace "are" with "have been" or use something like this:

Because no MATLAB versions exist under Jenkins Global Tool Configuration,

* If you want to go with the first sentence, is configure the right verb? How about register?

Also, I prefer under Jenkins to in Jenkins.

Reply:
Done. Chose the second suggestion.